pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -19,10 +19,12 @@
 , useFabricmanager ? false
 , ibtSupport ? false
 
-, prePatch ? ""
+, prePatch ? "" # why is this "" and not null ?
```

Comment:
I am not sure if its useful to commit this. We can look though history and maybe find out why its `""` otherwise just change to `null` and ask original author to review/comment

Reply:
From what I see `prePatch` and `postPatch` were introduced in 2 different commits by 2 different persons: 7b41780a0e4defe29e2bf9a17511b521db01c67d in 2017 adds `prePatch`, and then @Kiskae in 1e338049757a67c487babb604bc566daa59db52e in 2023 adds `postPatch`. So I suppose it is the same, will change my commit to just change `prePatch`'s default value to null

Reply:
I didn't touch it at the time to avoid rebuilds, since it would change all the builds that haven't specified a `prePatch` attribute.